cmd/crates/soroban-spec-tools/src/lib.rs

```diff
@@ -121,6 +121,9 @@ impl Spec {
             | ScType::I256
             | ScType::String
             | ScType::Bool => String::new(),
+            ScType::MuxedAddress => {
+                String::from("Can be public key (G13..) or a muxed account (M13..)")
+            }
             ScType::Address => String::from(
                 "Can be public key (G13..), a contract hash (6c45307) or an identity (alice), ",
```

Comment:
Oh wow, this is wrong from a long time ago:
```suggestion
                "Can be public key (G13..), a contract ID (C13...) or an identity (alice), ",
```